doc/classes/DisplayServer.xml

```diff
@@ -849,6 +849,37 @@
 				[b]Note:[/b] This method is implemented only on macOS.
 			</description>
 		</method>
+		<method name="ios_get_gesture_state" qualifiers="const">
+			<return type="int" enum="DisplayServer.IOSUIGestureState" />
+			<description>
+				Returns the state of the UI gestures currently in effect.
+				[b]Note:[/b] This method is only implemented on iOS.
```

Comment:
This note isn't necessary if it's named `ios_*`, but I agree that the naming is probably not necessarily right for all these at least